doc/mgr/restful.rst

```diff
@@ -40,15 +40,15 @@ can be generated with a command similar to::
 The ``restful.crt`` should then be signed by your organization's CA
 (certificate authority).  Once that is done, you can set it with::
 
-  ceph config set mgr mgr/restful/$name/crt -i restful.crt
-  ceph config set mgr mgr/restful/$name/key -i restful.key
+  ceph config-key set mgr mgr/restful/$name/crt -i restful.crt
```

Comment:
Should be "ceph config key set mgr/restful..." (no extra "mgr" bit between).

The extra "mgr" bit was new with "ceph config set", specifying which daemon the config setting applies to, whereas with "config-key" it's all just one global store.

Reply:
Fixed, thanks for the clarification!